### PR TITLE
Remove WASM from 23.1

### LINKED ISSUE #1547: Remove WASM from 23.1 docs
### Describe the Issue

https://github.com/redpanda-data/vtools/pull/1178 "removed all kinds of critical coproc pieces" for WASM from 23.1, so WASM is non-functional in 23.1 and needs to be removed from 23.1 docs.

See:
- https://redpandadata.slack.com/archives/C02FSLH3ZS9/p1682618748210919?thread_ts=1682609659.360009&cid=C02FSLH3ZS9
- https://redpandadata.slack.com/archives/C0246LPDYDU/p1682431151237039

### Updates to existing documentation

Remove WASM from 23.1 docs

### PR DESCRIPTION
**Pull request by shorgi**
Fixes #1547 

[Updated labs page](https://deploy-preview-1549--redpanda-documentation.netlify.app/docs/labs/)
- Because WASM exists in v22.3 and v22.2, and because it may return as a labs feature, this PR keeps the WASM pages in 23.1 but replaces their content with an admonition stating that the feature is not supported, and to use v22.3 or v22.2 if wanting to try the feature.

### CODE REVIEW

**Review by Feediver1 — APPROVED** [review]
lgtm...just occurred to me--for the headings/sections removed, if a user had those bookmarked, where will they land now? Redirects?

**Comment by shorgi** [comment]
> lgtm...just occurred to me--for the headings/sections removed, if a user had those bookmarked, where will they land now? Redirects?

In general, a link to a page with a heading (ww.foo.com/bar#heading) where the page exists but the heading is removed doesn't result in a 404 so there's no redirect needed. Only if the page is removed completely do redirects to it need updating. And this PR doesn't remove any pages, only headings and content.

File: docs/labs/index.mdx [hunk 1]
```diff
@@ -18,14 +18,6 @@ Redpanda labs are versions of Redpanda that are released for public experiments
...
-|:----|:----|
-|[Data transforms with WebAssembly](./data-transform)|v21.12.1-wasm-beta1|
-
-## Next steps
-
-[Install Redpanda labs](./install-preview) 
-
-
+:::important
+No labs features are supported in this release.
```
Comment (mattschumpert) [inline docs/labs/index.mdx:22]: @shorgi if there are no labs features why not just hide the whole labs section for now?

**Review by mattschumpert — APPROVED** [review]

File: docs/labs/index.mdx (hunk 1 above)
Comment (shorgi) [reply docs/labs/index.mdx:22]: @mattschumpert - there's a docs "dependency" from the 22.3 and 22.2 labs pages to the 23.1 page. Removing the 23.1 labs page would mess up the links from them to the latest version.

Comment (mattschumpert) [reply docs/labs/index.mdx:22]: k

**Pull request merged**